### DISCUSSION TITLE
Getting "This site can’t provide a secure connection" running DevGateway

### DISCUSSION THREAD

**Discussion by dhrumil83** [microsoft/Microsoft-Fabric-workload-development-sample#99] [Q&A] [answered]
Started seeing this a couple of days ago, but could be a Windows update, but checking to see if something changed on the Fabric side that might cause this.

Console
Microsoft.Fabric.Workload.DevGateway.exe
[2024-09-20 08:12:42.037] [Information] Starting DevGateway...
[2024-09-20 08:12:42.150] [Information] Getting user credentials...

URL: https://localhost:63300/?code=0.ARcAmC16Y3OsmEaqLSTsfVYPBf9go8UYdOdAhWNyvW439mNoASc.AgABBAIAAAApTwJmzXqdR4BN2miheQMYAwDs_wUA9P-B19i5YmRGkdQPNDOg2npkBYkC1RBShDjGdTp_37nBCyUzM5Qy8S0GsBJ8_2tnRLoUCXDZlIqha0p3EW7ywXVaUVG8uLSzPr3mWuymhI0TFoP9YafVDNfMfnPZp3tOMEOli1LrctfoLn7jhILXVRDXD4aRDemVepf__-2S1tLT0GV40LLWxyPBYciipFzHVPvbOGIzkGMAITlAaBUdYaJ0okA2nwmnGQ8u-sNdSJ9iCxLISoSIkCMkVPh-hS2emDAQfkSj1EwHxOB27x3HaL-Ijmnpx8Nc1GMjyyeH1MlLLqky7gTT7aJD05f-j8Ml476kD_furEo3A91a0LdDcmb8IGqsDYKnfBiGcGWdkfX31a_EktVGJ_BTGNCD6nF_KU_U8Gmy3SdNRTQMIpmwElLXXJk0ZDjsNULpJL797xPWr8a-ZbMIRs0YhATX4FQnKj86ZOyeXHkbA5DNHzzUyrPyTJH9HRfuV0XI658MdjuGByZcxQXDWpU2OsPnWUaRR1hjRQ6tzGaaJOlqbRB5_XWHSz5MZxgrwpyq5Ze0y7MjCO7ZGHOHmwpugHKynDuuMQeY9ZY4A-hZrvejRbF2ZhefyZLfoIzvc8f-ORJ7d0DQ5kaJgkOfYWIr1Ar0_hV3fGw-1ShFYtyNFFfgLEttw9aAiKDlH5lYOHTJsa1OMueKRByO6CQRxjQ6LyfRh7CCD3CdzkGzW6k3uJdbNU-rshGVJslLkANjaIzaly1NT9gPxGplkOlegJqRz_1xX8YxHoXMr1N3bd12ViBzhje_PoC1txnLy2eSjkKVKw4yzlQRYI1X_2-7VEeWmjZqZY69u_Z-x1PnEoXRL9wM6NRWn8Pg&client_info=eyJ1aWQiOiJiOTM0OTI2Yi0wZWJmLTRmNWEtYTVmNy03ZGJjNzZkOWU1MjciLCJ1dGlkIjoiNjM3YTJkOTgtYWM3My00Njk4LWFhMmQtMjRlYzdkNTYwZjA1In0&state=a9cc1cc9-bf95-4bbb-a8f7-e12a67594ca8d3aa4f45-85d2-4de9-9d28-481c92f22c51&session_state=b6e6be2a-510e-436a-b10d-f775382092a5

![Screenshot 2024-09-20 at 8 21 37 AM](https://github.com/user-attachments/assets/7a733a8f-a73d-4cc3-bef2-5e07fd12f31d)

**Comment by ramizsha** [comment]
Do you see any errors in the devgateway's console?

**Reply by dhrumil83** [reply]
Nothing in the devgateway console. Seems like it's waiting on the browser.

Should the localhost URL be http or https?

**Reply by dhrumil83** [reply, marked as answer]
Seems like it was a browser issue. The original redirect url was http but it was somehow going to https.
Used another browser and it works now.
